xls/examples/BUILD

```diff
@@ -562,6 +563,13 @@ place_and_route(
     placement_density = "0.8",
     synthesized_rtl = ":find_index_5000ps_model_unit_verilog_synth",
     target_die_utilization_percentage = "30",
+    density_fill_config = "fill.json"
+)
+
+gds_write(
+	name = "find_index_asic",
+	implemented_rtl = ":find_index_place_and_route",
+	klayout_lyt = "klayout.lyt",
```

Comment:
can we target asap7 instead?

Reply:
ASAP7 support in `bazel_rules_hdl` is not entirely working. It looks like it was tested only to the synthesis stage. When I tried to use synthesis results generated with this PDK in rules `place_and_route` and `gds_write` I encountered some problems, see https://github.com/hdl/bazel_rules_hdl/issues/172

Reply:
Please let me know if it would be OK for you to first merge this PR with the support for SKY130. Then once I solve the problems with ASAP7 I could open a follow up PR with ASAP7 examples.

Reply:
oh I realize that's probably outdated.